docs/execution-providers/CUDA-ExecutionProvider.md

```diff
@@ -38,6 +38,8 @@ ONNX Runtime built with cuDNN 8.x are not compatible with cuDNN 9.x.
 
 | ONNX Runtime             | CUDA   | cuDNN                                   | Notes                                                                                                                                                                                                                               |
 |--------------------------|--------|-----------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 1.18                     | 12.4   | 10.0.1.6 (Linux)<br/>10.0.1.6 (Windows) | The default CUDA version for ORT 1.18.To install CUDA 12 package, please look at [Install ORT](../install). Java CUDA 12 support is back for release 1.18                                                                             |
```

Comment:
This line makes it a bit confusing which CUDA version is the default for 1.18.

If it's CUDA 12, I think the notes column should say:

> The default CUDA version for ORT 1.18 is CUDA 12.4. To install CUDA 11 package, please look at [Install ORT](../install). Java CUDA 12 support is now available for release 1.18.

If it's CUDA 11.8, I think the notes column should say:

> The default CUDA version for ORT 1.18 is CUDA 11.8. To install CUDA 12 package, please look at [Install ORT](../install). Java CUDA 12 support is now available for release 1.18.